### PR TITLE
add a replace for kube-openapi in event-publisher-proxy

### PR DESCRIPTION
**Pull request by FriedrichWilken**
his pr adds a replace for `k8s.io/kube-openapi` to the latest commit to prevent security issues.

### CODE REVIEW

File: resources/eventing/values.yaml [hunk 1]
```diff
@@ -5,11 +5,11 @@ global:
   images:
     eventing_controller:
       name: eventing-controller
-      version: PR-14580
+      version: PR-14599
```
Comment (nachtmaar) [inline resources/eventing/values.yaml:8]: Is this a test ? :-D
Doesn't need to be updated but why not :-D

**Review by nachtmaar — APPROVED** [review]

File: resources/eventing/values.yaml (hunk 1 above)
Comment (FriedrichWilken) [reply resources/eventing/values.yaml:8]: uuups

**Review by nachtmaar — APPROVED** [review]

**Pull request merged**